less/lib/Dropdown.less

```diff
@@ -79,6 +79,49 @@
   left: auto;
   right: 0;
 }
+.Dropdown-menu--below {
+    padding-left: 10px !important;
+    padding-top: 10px !important;
+    bottom: inherit !important;
+    position: absolute !important;
+    top: 100% !important;
+    overflow: auto;
+    max-height: 45vh !important;
+    min-width: 189px !important;
+    z-index: 1000;
+    float: left;
+    background-color: #fff;
+    -webkit-background-clip: padding-box;
+    background-clip: padding-box;
+    border: 1px solid rgba(0,0,0,.15);
+    -webkit-transform: none !important;
+    transform: none !important;
+    -webkit-transition: none !important;
+    -moz-transition: none !important;
+    -o-transition: none !important;
+    transition: none !important;
+    > li > div {
```

Comment:
@franzliedke we have mixins which automatically apply browser prefixes. See less/lib/mixins/vendor-prefixes.less